src/components/ArticleView/ArticleDetail/Category.tsx

```diff
@@ -7,19 +7,21 @@ interface Props {
 }
 
 const Category = ({ category }: Props) => {
-  const categories: { [key: string]: string } = {
-    마케팅: 'red',
-    디자인: 'blue',
-    기획: 'purple',
-    개발: 'yellow',
+  const categories: { [key: string]: Array<string> } = {
+    marketing: ['red', '마케팅'],
+    design: ['blue', '디자인'],
+    plan: ['purple', '기획'],
+    develop: ['yellow', '개발'],
   };
```

Comment:
```suggestion
  const categories: { [key: string]: Array<string> } = {
    marketing: {
      color: 'red', title: '마케팅',
    },
    design: {
      color: 'blue', title: '디자인',
    },
    plan: {
      color: 'purple', title: '기획',
    },
    develop: {
      color: 'yellow', title: '개발',
    },
  };
```

지금처럼 배열을 value로 해서 index를 참조하여 사용하는 방법도 있지만,
객체를 사용하면 가독성 측면에서 더 좋지 않을까요 ?

Reply:
넵 그러네요! 형조님이 작성하신 컴포넌트 참조해서 아이콘도 반영되게 변경했습니다!